src/components/GameGrid/GameBoard.tsx

```diff
@@ -32,11 +37,46 @@ const GameBoard: React.FC<Props> = ({ gameData }: Props) => {
       <TextContainer>
         <Shout>Other games</Shout>
 
-        <Loud>Knockout rounds</Loud>
+        <Loud>Round of 32</Loud>
         <Paragraph>Knockout rounds begin on Friday 15 January and end on Saturday 30&nbsp;January.</Paragraph>
       </TextContainer>
 
       <KnockoutRounds knockoutRoundData={gameData.knockout} />
+
+      <TextContainer>
+        <Loud>Round of 16</Loud>
+        <Paragraph></Paragraph>
+      </TextContainer>
+
+      <GroupStages groupStageData={gameData.groupStages} />
+
+/**   <TextContainer>
+        <Loud>Quarter Finals</Loud>
+        <Paragraph></Paragraph>
+      </TextContainer>
+
+      <QuarterFinals quarterFinalData={gameData.quarterFinal} />
+
+      <TextContainer>
+        <Loud>Semi Finals</Loud>
+        <Paragraph></Paragraph>
+      </TextContainer>
+
+      <SemiFinals semiFinalData={gameData.semiFinal} />
+
+      <TextContainer>
+        <Loud>3rd Place</Loud>
+        <Paragraph></Paragraph>
+      </TextContainer>
+
+      <RunnerUp RunnerUpData={gameData.runnerUp} />
+
+      <TextContainer>
+        <Loud>Final</Loud>
+        <Paragraph></Paragraph>
+      </TextContainer>
+
+      <Finals FinalData={gameData.final} /> */
```

Comment:
Despite what GitHub tells you, this isn't actually commented out!
It also crashes the site - Final and RunnerUp only take a single `SinglePoll`, rather than a `Record` (i.e. `{ 0: SinglePoll, 1: SinglePoll }` etc), bu the components are setup for a `Record`
```suggestion
{/**   <TextContainer>
        <Loud>Quarter Finals</Loud>
        <Paragraph></Paragraph>
      </TextContainer>

      <QuarterFinals quarterFinalData={gameData.quarterFinal} />

      <TextContainer>
        <Loud>Semi Finals</Loud>
        <Paragraph></Paragraph>
      </TextContainer>

      <SemiFinals semiFinalData={gameData.semiFinal} />

      <TextContainer>
        <Loud>3rd Place</Loud>
        <Paragraph></Paragraph>
      </TextContainer>

      <RunnerUp RunnerUpData={gameData.runnerUp} />

      <TextContainer>
        <Loud>Final</Loud>
        <Paragraph></Paragraph>
      </TextContainer>

      <Finals FinalData={gameData.final} /> */}
```